### PR TITLE
update elixir version

### PR DESCRIPTION
**Pull request by ayrat555**
Compilation fails on my linux desktop using elixir 1.6. Let's see if fails on circle ci.

By the way, I also tested compilation on different versions of elixir using docker. Project compiles in elixir 1.3-1.5, but fails in elixir 1.6

Update:
Project compiles with omisego fork of libsecp256k1

- update elixir version
- fix tests

Actually I was trying to run ex_wire project, but it did not compile because of `libsecp256k1` dependency in blockchain project

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -40,15 +41,15 @@ defmodule Blockchain.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:libsecp256k1, "~> 0.1.2"},
+      {:libsecp256k1, github: "omisego/libsecp256k1"},
```
Comment (hayesgm) [inline mix.exs]: You cannot publish a package to Hex with a github dep

Comment (ayrat555) [reply mix.exs]: yep. I wait for libsecp256k1 update.

Comment (hayesgm) [reply mix.exs]: Here ya go! https://hex.pm/packages/libsecp256k1/0.1.4

**Comment by ayrat555** [comment]
@hayesgm btw I'm sorry, Geoff, but I should have bumped the version of blockchain and it should be published on hex.pm. Can you do it?

**Comment by hayesgm** [comment]
@ayrat555 https://hex.pm/packages/blockchain/0.1.7 :)

**Comment by ayrat555** [comment]
Thank you!

**Pull request merged**